src/libraries/System.Private.CoreLib/src/System/AppContext.cs

```diff
@@ -135,6 +135,37 @@ public static void SetSwitch(string switchName, bool isEnabled)
             }
         }
 
+#if NETCOREAPP
```

Comment:
This ifdef should not be needed. AppContext.cs is only ever compiled into CoreLib that has `NETCOREAPP` defined.

Reply:
I was trying to solve the problem with browser, where it seems `RuntimeEventSource` doesn't exist -- but maybe this isn't the right ifdef...